packages/flame_3d/lib/src/extensions/color.dart

```diff
@@ -3,5 +3,10 @@ import 'dart:ui';
 
 extension ColorExtension on Color {
   /// Returns a Float32List that represents the color as a vector.
-  Float32List get storage => Float32List.fromList([a, r, g, b]);
+  Float32List get storage => Float32List.fromList([
+        opacity,
+        red.toDouble() / 255,
+        green.toDouble() / 255,
+        blue.toDouble() / 255,
+      ]);
```

Comment:
How come this changed? :thinking:

Reply:
I am reverting the color changes to the old version